src/themes/teams/components/Chat/chatMessageStyles.ts

```diff
@@ -16,17 +17,24 @@ const chatMessageStyles: ComponentSlotStylesInput<ChatMessageProps, ChatMessageV
     maxWidth: v.messageWidth,
     wordBreak: 'break-word',
     wordWrap: 'break-word',
+    ':focus': {
+      outline: 'none',
+      messageBody: {
```

Comment:
this won't work - `messageBody` styles block won't be applied on root focus

Reply:
I committed that by mistake, reverted but you already say it haha